### ISSUE TITLE
Azure.AI.OpenAI 2.0.0-beta.3 (Unable to parse version value '2.0.0-beta.3' from 'packages.config')

### ISSUE THREAD

**Issue by limdv** [Azure/azure-sdk-for-net#45718]
### Library name and version

Azure.AI.OpenAI 2.0.0-beta.3

### Query/Question

Hello,

After I installed Azure.AI.OpenAI 2.0.0-beta.3 I receive the following build errors. Anyone else seen and been able to resolve this issue. I can't seem to find any information regarding this error and it may not be specific to this package. It seems to be any package with 4 segments in the version. Any idea how to fix this?

>C:\Projects\SMTMVC\.nuget\nuget.targets(92,9): error : Unable to parse version value '2.0.0-beta.3' from 'packages.config'.
3>C:\Projects\SMTMVC\.nuget\nuget.targets(92,9): error MSB3073: The command ""C:\Projects\SMTMVC\.nuget\NuGet.exe" install "C:\Projects\SMTMVC\SMTMVC\packages.config" -source ""  -NonInteractive -RequireConsent -solutionDir "C:\Projects\SMTMVC\ "" exited with code 1.

### Environment

MVC project targeting the .NET Framework 4.8 in Visual Studio 2022 (Version 17.11.0), Nuget Package Manager (Version 6.11.0).

**Comment by jsquire** [comment]
Hi @limdv.  Thanks for reaching out and we regret that you're experiencing difficulties.  The Azure SDK packages have no insight nor influence over NuGet nor the process that your environment uses to bump package reference versions.  As a result, the Azure SDK team is unable to assist.

From what I was able to determine, it looks as if the NuGet repository would be the best place to seek help.  I've transferred this there as [#13750](https://github.com/NuGet/Home/issues/13750).

**Comment by limdv** [comment]
Jesse,
Thanks for your reply and help redirecting this for me.
Thanks,Darren Lim


    On Wednesday, August 28, 2024 at 08:40:58 PM CDT, Jesse Squire ***@***.***> wrote:  
 
 


Hi @limdv. Thanks for reaching out and we regret that you're experiencing difficulties. The Azure SDK packages have no insight nor influence over NuGet nor the process that your environment uses to bump package reference versions. As a result, the Azure SDK team is unable to assist.

From what I was able to determine, it looks as if the NuGet repository would be the best place to seek help. I've transferred this there as #13750.
